src/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -22,14 +22,20 @@
 
 import android.accounts.Account;
 import android.app.Activity;
+import android.content.Context;
 import android.content.Intent;
 import android.content.SharedPreferences;
+import android.database.Cursor;
+import android.graphics.Bitmap;
+import android.net.Uri;
 import android.os.AsyncTask;
 import android.os.Bundle;
 import android.os.Environment;
 import android.preference.PreferenceManager;
+import android.provider.MediaStore;
 import android.support.v4.app.DialogFragment;
 import android.support.v7.app.ActionBar;
+import android.util.Log;
```

Comment:
Import not needed, you can use the option Code => Optimize Imports once you have finished your code or from time to time to delete automatically the unused imports